python/mxnet/context.py

```diff
@@ -3,7 +3,17 @@
 from __future__ import absolute_import
 
 class Context(object):
-    """Constructing a context.
+    """Constructs a context.
+
+    MXNet can run operations on CPU and different GPUs.
+    A context describes the device type and id on which computation should be carried on.
```

Comment:
should we say "ID" (capitalized is the more common shorthand for "identity" unless there's some precedent in the code for using the lower case)